05_Customizing_OrbitDB/01_Definitions.md

```diff
@@ -0,0 +1,58 @@
+# What are Stores, AccessControllers and Indicies.
+
+Before we start to implement
+our custom store, access controller and index
+to allow our users to comment on each others
+notes, we should probably define what a store,
+AccessController and Index are and what
+function they have in OrbitDB.
+
+## The Store
+You have already worked with several
+Stores in the Tutorial.
+The `docstore` was used to store
+the individual pieces (or rather their CIDs)
+and the `kvstore` type was used to
+represent a user.
+
+Both of those were instances of a Store.
+They stored some data and provided easy
+functions to access them, like `get`, `put` or `set`,
+with which you could easily interact with
+the database itself.
+
+## The Index
+But a store doesn't actually
+store the current state of the database,
+because this is done in the `Index`.
+
+The index parses the log of operations
+generated by the store while
+modifying the database and generates
+an easy to access and use representation
+of the current state of the database.
+
+The index is then used by the Store
+to implement it's API.
+
+## The Access Controllers
+Access Controllers or short ACL (Access Control List)
+were already discussed [02: Managing Data](../01_Tutorial/02_Managing_Data).
+
+They are a piece of code, that is invoked, whenever
+you try to write to the database, to ensure,
+that you have the right to do so.
+
+It is setup when you create a database
+and stored in the Manifest of the database.
+(That thing, that the mulithash in the middle
+of your orbitdb address refers to)
+
+Anyone, who supports this type of access
+controller, can follow the rules of the access controller.
+
+But this also means, you cannot modify your access
+controller and it's options after you created
+the database.
```

Comment:
This is true in the case of the IPFS AC but not the OrbitDB AC and others. Not sure how to clarify this at this point in the chapter :thinking:

Reply:
What I tried to explain here, that I cannot change from the OrbitDB AC to the IPFS AC or to some custom AC, 
after the Database has been created.